doc/_static/javascript/index.js

```diff
@@ -33,7 +33,7 @@ window.addEventListener ('load', function () {
         $faqlinks = $('a.reference.internal[href^="FAQ.html#"]')
     }
     $faqlinks.each (function () {
-        this.innerText = this.innerText.split ('?')[0] + '?';
+        this.parentNode.parentNode.remove ()
```

Comment:
What is this line for?

Reply:
to remove the links for the faq questions from the sidebar (since they are too long and not needed)